### PR TITLE
Update cosmos-hub-roadmap-2.0.md Liquidity

### PR DESCRIPTION
**Pull request by okwme**

### CODE REVIEW

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ The upgrades aim to add features such as liquidity, economic security, usability
 - Gaia v7.0.x
 - Cosmos SDK v0.45
   - Minimal update with small fixes
+- Liquidity v1.4.4
```
Comment (nooomski) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md]:
```suggestion
- Gravity DEX: Liquidity v1.4.5
```

**Review by nooomski — APPROVED** [review]
Add Gravity DEX

File: docs/roadmap/cosmos-hub-roadmap-2.0.md (hunk 1 above)
Comment (dongsam) [reply docs/roadmap/cosmos-hub-roadmap-2.0.md]: Hi, Thanks for the update, cosmos-sdk v0.45.0 based Liquidty is in progress at https://github.com/Gravity-Devs/liquidity/pull/10, which is expected to be v1.4.5.

Comment (okwme) [reply docs/roadmap/cosmos-hub-roadmap-2.0.md]: thanks for clarification @dongsam!

**Pull request merged**